life_expectancy/load_save.py

```diff
@@ -0,0 +1,20 @@
+''' Clean data code'''
+import argparse
+from pathlib import Path # pylint: disable=import-error
```

Comment:
```suggestion
from pathlib import Path
```

Algum outra cena pode estar a causar este import error